packages/wonder-blocks-testing/package.json

```diff
@@ -21,7 +21,9 @@
         "@khanacademy/wonder-stuff-testing": "^0.0.2",
         "@storybook/addon-actions": "^6.4.8",
         "node-fetch": "^2.6.7",
-        "react": "16.14.0"
+        "aphrodite": "^1.2.5",
+        "react": "16.14.0",
+        "react-router-dom": "5.3.0"
```

Comment:
The harness and the adapters reference these peer deps that we already use in other parts of Wonder Blocks.